lua/nvim-treesitter-textobjects/shared.lua

```diff
@@ -447,4 +472,28 @@ function M.check_support(bufnr, query_group, queries)
   return true
 end
 
+M.range6_range4 = range6_range4
```

Comment:
Avoid unnecessary wrappers (makes it harder to use LSP for checking references); just expose the original function.